include/kernels/laplace.cuh

```diff
@@ -159,7 +161,8 @@ namespace quda
                                                                          active);
       break;
     default:
-      errorQuda("Unsupported laplace direction %d given", arg.dir);
+      applyLaplace<Float, nDim, nColor, nParity, dagger, kernel_type, -1>(out, arg, coord, x_cb, parity, idx,
```

Comment:
Can we just do something like:
```C++
case 3:
  apply < ...., 3>... 
  break;
case 4:
default:
  apply <...., -1>
```

here?

Reply:
Yes, that's much neater.